src/js/background.js

```diff
@@ -1,172 +1,193 @@
-import * as constants from './constants.js';
```

Comment:
add this line back, removing it caused the reference error on chrome

Reply:
that must have been a mistake anyway added that back